utils/log.js

```diff
@@ -5,85 +5,49 @@ const chalk = require('chalk');
 /**
  * Logs a simple message to the console
  * @param {string} message Message content
+ * @param {object} [options]
+ * @param {boolean} [options.newLine=false]
+ * @param {boolean} [options.indent=false]
+ * @param {'info'|'success'|'error'} [options.level='info']
```

Comment:
Wouldn't be better to use constants for the level param 🤔

Reply:
True, but because we are using typecheck it will fail if you misspell some word :)

Reply:
But does it helps to autocomplete the param?

Reply:
Thats true. I'm changing that